pkg/moov/disputes.go

```diff
@@ -135,3 +170,47 @@ func (c Client) GetDispute(ctx context.Context, disputeID string) (*Dispute, err
 
 	return CompletedObjectOrError[Dispute](resp)
 }
+
+// AcceptDispute Accepts a dispute.
```

Comment:
Nit: Is there more to explain here? This reads like you're using the word to define the word.

Reply:
Sure, i was taking from the official text on the webpage but i added a little more